build.make:69: *** missing separator.  Stop.

Trying to compile libArcus on OSX 10.11.6 and getting this error.

```
$: cmake .. -DPYTHON_LIBRARY=/usr/local/Cellar/python3/3.5.2/Frameworks/Python.framework/Versions/3.5/lib/libpython3.5m.dylib

-- Found SIP version: Unable to import sipconfig and determine your sip configuration.

-- Configuring done
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   Arcus

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done
-- Build files have been written to: /Users/user/projects/CuraEngine/libArcus/build
```

**Error:**

```
make
[ 27%] Built target Arcus
CMakeFiles/python_module_Arcus.dir/build.make:69: *** missing separator.  Stop.
make[1]: *** [CMakeFiles/python_module_Arcus.dir/all] Error 2
make: *** [all] Error 2
```

Sure here it is:

```
$: make VERBOSE=1
/usr/local/Cellar/cmake/3.6.1/bin/cmake -H/Users/user/Projects/nebula/CuraEngine/libArcus -B/Users/user/Projects/nebula/CuraEngine/libArcus/build --check-build-system CMakeFiles/Makefile.cmake 0
/usr/local/Cellar/cmake/3.6.1/bin/cmake -E cmake_progress_start /Users/user/Projects/nebula/CuraEngine/libArcus/build/CMakeFiles /Users/user/Projects/nebula/CuraEngine/libArcus/build/CMakeFiles/progress.marks
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f CMakeFiles/Makefile2 all
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f CMakeFiles/Arcus.dir/build.make CMakeFiles/Arcus.dir/depend
cd /Users/user/Projects/nebula/CuraEngine/libArcus/build && /usr/local/Cellar/cmake/3.6.1/bin/cmake -E cmake_depends "Unix Makefiles" /Users/user/Projects/nebula/CuraEngine/libArcus /Users/user/Projects/nebula/CuraEngine/libArcus /Users/user/Projects/nebula/CuraEngine/libArcus/build /Users/user/Projects/nebula/CuraEngine/libArcus/build /Users/user/Projects/nebula/CuraEngine/libArcus/build/CMakeFiles/Arcus.dir/DependInfo.cmake --color=
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f CMakeFiles/Arcus.dir/build.make CMakeFiles/Arcus.dir/build
make[2]: Nothing to be done for `CMakeFiles/Arcus.dir/build'.
[ 27%] Built target Arcus
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f CMakeFiles/python_module_Arcus.dir/build.make CMakeFiles/python_module_Arcus.dir/depend
CMakeFiles/python_module_Arcus.dir/build.make:69: *** missing separator.  Stop.
make[1]: *** [CMakeFiles/python_module_Arcus.dir/all] Error 2
make: *** [all] Error 2
```

--- a/cmake/FindSIP.cmake
+++ b/cmake/FindSIP.cmake
@@ -40,8 +40,10 @@ ELSE(SIP_VERSION)
   FIND_FILE(_find_sip_py FindSIP.py PATHS ${CMAKE_MODULE_PATH})
 
   SET(ENV{PYTHONPATH} ${CMAKE_INSTALL_PREFIX}/${PYTHON_SITE_PACKAGES_DIR})
-  EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} ${_find_sip_py} OUTPUT_VARIABLE sip_config)
-  IF(sip_config)
+  EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} ${_find_sip_py}
+                  OUTPUT_VARIABLE sip_config
+                  RESULT_VARIABLE sip_config_returncode)
+  IF(sip_config_returncode EQUAL 0)
     STRING(REGEX REPLACE "^sip_version:([^\n]+).*$" "\\1" SIP_VERSION ${sip_config})
     STRING(REGEX REPLACE ".*\nsip_version_num:([^\n]+).*$" "\\1" SIP_VERSION_NUM ${sip_config})
     STRING(REGEX REPLACE ".*\nsip_version_str:([^\n]+).*$" "\\1" SIP_VERSION_STR ${sip_config})
@@ -49,7 +51,7 @@ ELSE(SIP_VERSION)
     STRING(REGEX REPLACE ".*\ndefault_sip_dir:([^\n]+).*$" "\\1" SIP_DEFAULT_SIP_DIR ${sip_config})
     STRING(REGEX REPLACE ".*\nsip_inc_dir:([^\n]+).*$" "\\1" SIP_INCLUDE_DIR ${sip_config})
     SET(SIP_FOUND TRUE)
-  ENDIF(sip_config)
+  ENDIF(sip_config_returncode EQUAL 0)
 
   IF(SIP_FOUND)
     IF(NOT SIP_FIND_QUIETLY)

